app/models/challenge.rb

```diff
@@ -42,6 +42,23 @@ def featured_contributions
     end
   end
 
+  def latest_contributions
+    ordering = "published_at DESC"
+
+    case active_stage
+    when 'experience'
+      experience_stage.experiences.published.order_by(ordering).first(2)
```

Comment:
This PR looks good! My only concern here is when / how we implemented `order_by`...

Essentially when you call `unscoped` it will undo ALL other scopes you've applied to this query. I learned that the better way to do this is to actually do a `reorder` call so that it doesn't eliminate the scopes for `published` etc.: http://apidock.com/rails/ActiveRecord/QueryMethods/reorder

We may also want to investigate where else we are using that so we don't accidentally show any unpublished items throughout the app (other than a users' show page).